pkg/client/v1/auth.go

```diff
@@ -85,23 +89,27 @@ type ServerResponse struct {
 	StatusCode int
 }
 
-func (c Client) get(ctx context.Context, endpoint string, resp interface{}) error {
-	request, err := newGetRequest(ctx, c.url, endpoint)
+func (c Client) get(ctx context.Context, endpoint string, query url.Values, resp interface{}) error {
+	request, err := newGetRequest(ctx, c.url, endpoint, query)
 	if err != nil {
 		return err
 	}
 
 	return c.do(request, &resp)
 }
 
-func newGetRequest(ctx context.Context, uri, endpoint string) (*http.Request, error) {
+func newGetRequest(ctx context.Context, uri, endpoint string, query url.Values) (*http.Request, error) {
 	u, err := url.Parse(uri)
 	if err != nil {
 		return nil, err
 	}
 
 	u.Path = path.Join(apiVersion, endpoint)
 
+	if len(query) > 0 {
+		u.Path = fmt.Sprintf("%s?%s", u.Path, query.Encode())
```

Comment:
```suggestion
		u.RawQuery = query.Encode()
```

Shouldn't be formatting the string manually. Let url deal with that.